frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -178,6 +179,76 @@ export const surveyLogic = kea<surveyLogicType>([
                 editingSurvey: (_, { editing }) => editing,
             },
         ],
+        survey: [
+            { ...NEW_SURVEY } as NewSurvey | Survey,
+            {
+                setDefaultForQuestionType: (state, { type }) => {
+                    if (type === SurveyQuestionType.Open) {
+                        return {
+                            ...state,
+                            questions: [
+                                {
+                                    ...state.questions[0],
+                                    question: 'Give us feedback on our product!',
+                                },
+                            ],
+                        }
+                    } else if (type === SurveyQuestionType.Link) {
+                        return {
+                            ...state,
+                            questions: [
+                                {
+                                    ...state.questions[0],
+                                    question: 'Do you want to join our upcoming webinar?',
+                                },
+                            ],
+                            appearance: {
+                                ...state.appearance,
+                                submitButtonText: 'Register',
```

Comment:
It seems we need to remove these from other cases, because otherwise it borks things:

![2023-09-20 13 01 53](https://github.com/PostHog/posthog/assets/7115141/3e452618-15dc-49d2-848b-70b8525ab299)

Also, we shouldn't override what the user has set, so I think `state.appearance` should come later here, otherwise it gets annoying if the background color keeps changing because they changed the question type